### DISCUSSION TITLE
Remove client-side validations

### DISCUSSION THREAD

**Discussion by bennypowers** [RedHat-UX/red-hat-design-system#1636] [Ideas]
Elements like CTA, Footer, Navigation and others contain client side validations which log warnings via `Logger` whenever they find invalid HTML children.

For example, this (incorrect) HTML 
```html
<rh-cta>
  <p><a></a></p>
</rh-cta>
```

Will produce this message:

> The first child in the light DOM must be a supported call-to-action tag (&lt;a>, &lt;button>)

## The Problem
This means we're shipping validation code to pages which don't need it
and we're also logging developer-facing messages on customer-facing sites

## Proposed Solution
I propose we remove *all* of these validations, and instead use a tool like @paceaux's [Selector Hound](https://github.com/paceaux/selector-finder) or other extensible HTML validators to run validations out-of-band

**Comment by paceaux** [comment]
I've got an issue open on Selector Hound to see how we can work on this. Working from a manifest or a custom element registry may be a novel way to do this.

**Reply by bennypowers** [reply]
https://github.com/paceaux/selector-finder/issues/39
